docs/essentials/index.mdx

```diff
@@ -1,91 +1,39 @@
-| `@storybook/addon-docs`        | `csfPluginOptions` | Provides additional configuration for Storybook's CSF plugin. Can be disabled with `null`.                                                                     |
-|                                | `mdxPluginOptions` | Provides additional configuration options and plugin configuration for [MDX documentation](../writing-docs/mdx.mdx#markdown-tables-arent-rendering-correctly). |
-| `@storybook/addon-controls`    | N/A                | N/A                                                                                                                                                            |
-| `@storybook/addon-backgrounds` | N/A                | N/A                                                                                                                                                            |
-| `@storybook/addon-toolbars`    | N/A                | N/A                                                                                                                                                            |
-| `@storybook/addon-measure`     | N/A                | N/A                                                                                                                                                            |
-
-When you start Storybook, your custom configuration will override the default.
+Many of the features above can be configured via [parameters](../writing-stories/parameters.mdx). See each feature's documentation (linked above) for more details.
 
-### Disabling addons
+### Disabling features
 
-If you need to disable any of the Essential's addons, you can do it by changing your [`.storybook/main.js`](../configure/index.mdx#configure-story-rendering) file.
+If you need to disable any of the essential features, you can do it by changing your [`.storybook/main.js|ts`](../configure/index.mdx#configure-your-storybook-project) file.
 
-For example, if you wanted to disable the [backgrounds addon](./backgrounds.mdx), you would apply the following change to your Storybook configuration:
+For example, if you wanted to disable the [backgrounds feature](./backgrounds.mdx), you would apply the following change to your Storybook configuration:
 
 {/* prettier-ignore-start */}
 
-<CodeSnippets path="storybook-main-disable-addon.md" />
+<CodeSnippets path="essential-feature-disable.md" />
 
 {/* prettier-ignore-end */}
 
 <Callout variant="info" icon="💡">
-  You can use the following keys for each individual addon: `actions`, `backgrounds`, `controls`, `docs`, `viewport`, `toolbars`, `measure`, `outline`, and `highlight`.
+  You can use the following keys for each individual feature: `actions`, `backgrounds`, `controls`, `highlight`, `measure`, `outline`, `toolbars`, and `viewport`.
```

Comment:
When you can, include interactions as well as it seems that's now possible to disable it per #31146. You can see it [here](https://github.com/storybookjs/storybook/blob/next/code/core/src/core-server/presets/common-preset.ts#L214), also the features [documentation](https://storybook.js.org/docs/api/main-config/main-config-features) should also feature it and the other options, wouldn't you agree?